sklearn/datasets/svmlight_format.py

```diff
@@ -77,6 +78,7 @@ def load_svmlight_file(f, n_features=None, dtype=np.float64,
         bigger sliced dataset: each subset might not have examples of
         every feature, hence the inferred shape might vary from one
         slice to another.
+        n_features is required is offset or length are passed a default values.
```

Comment:
required i**f**
are passed a default value (i.e. no s)
I am not sure I understand what you mean by "are passed a default value" to be honest.